chart/templates/jobs/cache-maintenance/job.yaml

```diff
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2022 The HuggingFace Authors.
+
+{{- if .Values.images.jobs.cacheMaintenance }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  labels: {{ include "labels.cacheMaintenance" . | nindent 4 }}
+  name: "{{ include "name" . }}-job-cache-maintenance"
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": post-upgrade
```

Comment:
not needed after an install ?

Reply:
Yes, I think that we could run it after an install to populate the cache database. So, post-install too, maybe.

Note that a next step is to change it to a cron job, to have more control on the time it is run.

Reply:
I fixed it on https://github.com/huggingface/datasets-server/pull/1017/commits/e67005f028c2117e44c80a88f7d170ab664afaef thanks!